smart_card_connector_app/src/testing-smart-card-simulation-constants.js

```diff
@@ -27,6 +27,18 @@ const GSC = GoogleSmartCard;
 GSC.TestingLibusbSmartCardSimulationConstants.CPP_HELPER_NAME =
     'SmartCardConnectorApplicationTestHelper';
 
+// Names of the device types defined by TestingSmartCardSimulation.
```

Comment:
nit: I suspect that these strings need to match the ones in testing_smart_card_simulation.cc - if so, it would be good to add that to the comment.

Reply:
done, thanks!